x-pack/filebeat/module/iptables/log/config/input.yml

```diff
@@ -20,38 +20,6 @@ publisher_pipeline.disable_host: {{ inList .tags "forwarded" }}
 
 processors:
   - add_locale: ~
-{{ if .community_id }}
```

Comment:
Did you mean to remove the community_id processor logic?  I think that needs to stay because others may be using it.

Reply:
I moved the community_id processor to the pipeline, now line 201. It didn't make sense to have it in filebeat as it was doing processing that was just repeated in the grok in the pipeline, so i moved it and deleted the duplicate processing.

Reply:
sorry, missed that.  OK, then we should probably remove the following from `x-pack/filebeat/module/iptables/log/manifest.yml`

```
  - name: community_id
    default: true
```
Since the variable won't do anything anymore.

Reply:
if someone has that set after its removed, will it cause a fatal error or just not do anything?

Reply:
removed.